test/codegen/value_integrity_test.cpp

```diff
@@ -68,6 +68,7 @@ void DivideByZeroTest(const codegen::type::Type &data_type, ExpressionType op) {
 
   typedef void (*func)(CType);
   func f = (func)code_context.GetRawFunctionPointer(function.GetFunction());
+
   EXPECT_THROW(f(0), DivideByZeroException);
```

Comment:
You may also want to check the error message in this exception. This is what this PR is trying to do.